iap/README

```diff
@@ -1,4 +0,0 @@
-How to package the IAP extension with your application:
-
-  cd $(dirname $(which make_apk.py))
-  python make_apk.py --name=iap --package=org.xwalk.extensions.iap --permissions=iap --extensions="/<path-to-xwalk-android-extension>/out/Default/gen/iap/" --arch=x86 --manifest=/<path-to-xwalk-android-extension>/examples/iap/iap.json
```

Comment:
This content of the README should be for the sample project, not the extension, so I removed it and will add to the sample.